library/Theme/Customizer.php

```diff
@@ -5,29 +5,34 @@ class Customizer
 {
     public function __construct()
     {
-        add_action('after_setup_theme', array($this, 'customizerHeader'), 5);
+        add_action('after_setup_theme', [$this, 'enableCustomizer'], 5);
+        add_filter('kirki/config', [$this, 'configureKirki'], 20);
     }
 
-    public function customizerHeader()
+    public function enableCustomizer()
     {
-        add_filter(
-            'Municipio/Controller/BaseController/Customizer',
-            array($this, 'activateCustomizerFeatures')
-        );
+        $enable = function ($boolean) {
+            return true;
+        };
 
-        add_filter(
-            'Municipio/Theme/Enqueue/Bem',
-            array($this, 'activateCustomizerFeatures')
-        );
-
-        add_filter(
-            'Municipio/Widget/Widgets/CustomizerWidgets',
-            array($this, 'activateCustomizerFeatures')
-        );
+        add_filter('Municipio/Controller/BaseController/Customizer', $enable);
+        add_filter('Municipio/Theme/Enqueue/Bem', $enable);
+        add_filter('Municipio/Widget/Widgets/CustomizerWidgets', $enable);
```

Comment:
Note, since these are filters, they _probably_ shouldn't be added within a filter. They should instead be registered as soon as possible to prevent them from being lost added too late. That's for a different PR though.

Reply:
-> @rasjostrom

These filters may be related to why [test-latest](https://github.com/simrishamn/Municipio/tree/test-latest) loses the menu.